src/staticanalysis/bot/static_analysis_bot/report/base.py

```diff
@@ -143,8 +144,11 @@ def pluralize(word, nb):
             extras_comments=extras,
             defects_total=pluralize('defect', nb),
             defects='\n'.join(defects),
-            analyzers='\n'.join(analyzers),
         )
+
+        if analyzers:
+            comment += '\n' + COMMENT_RUN_ANALYZERS.format(analyzers='\n'.join(analyzers))
```

Comment:
Looks like `\n` wasn't necessary, as the test is failing because there are too many `\n` characters.

Reply:
Now it shouldn't fail anymore.